packages/changeset-form/addon/components/changeset-form/context/index.hbs

```diff
@@ -0,0 +1,15 @@
+{{yield
```

Comment:
I think you need a correspondent TS/Js file here, I'm not sure if template-only components work in addons.

Reply:
I think we are good, reference: https://discuss.emberjs.com/t/can-addons-use-template-only-components/17399/6?u=betocantu93